training/general/src/environments/sklearn-1.1-ubuntu20.04-py38-cpu/context/README.md

```diff
@@ -0,0 +1,35 @@
+# SKLearn Docker Environment 
```

Comment:
change folder name to be <framework>-<framework version>-<gpu driver>-<driver version>
@cassieesvelt is working on updating env name for all images and can provide the exact name for sklearn.